src/server/db_slice.cc

```diff
@@ -495,6 +495,43 @@ bool DbSlice::Del(DbIndex db_ind, PrimeIterator it) {
   return true;
 }
 
+void DbSlice::FlushSlotsFb(const absl::flat_hash_set<SlotId>& slot_ids) {
```

Comment:
If `flat_hash_set<SlotId>` is being passed a lot I recommend introducing an alias